src/model_filereading.cpp

```diff
@@ -617,70 +230,29 @@ bool Model::getSymmetryElements(std::string group, std::vector<int> &sym_matrix_
 // METHOD DEFINITIONS //
 ////////////////////////
 
-// returns the radius of an atom with a given symbol
-double Model::findRadiusOfAtom(const std::string& symbol){
-  return _radius_map[symbol];
+// Returns the radius of an atom with a given symbol
+// If the symbol is not intially found, the function strips everything after the first
+// non-letter character and tries to look up the radius again. If the symbol is still not
+// found, defaults to 0
+double Model::findRadiusOfAtom(std::string symbol) const {
+  if (_radius_map.count(symbol)){
+    return _radius_map.at(symbol);
+  }
+
+  symbol = ImportMngr::stripCharge(symbol);
+
+  if (_radius_map.count(symbol)){
+    return _radius_map.at(symbol);
+  }
+  
+  return 0;
 }
 
-double Model::findRadiusOfAtom(const Atom& at){
+double Model::findRadiusOfAtom(const Atom& at) const {
   return findRadiusOfAtom(at.symbol);
 }
 
 double Model::findWeightOfAtom(const std::string& symbol){

```

Comment:
Might be useful to add a flag in case an element weight is not found to show a warning in the report under the molar weight.